code/hsec-tools/src/Security/Advisories/Parse.hs

```diff
@@ -102,7 +97,9 @@ parseAdvisory policy attrs raw = do
   (frontMatter, rest) <- first MarkdownFormatError $ advisoryDoc markdown
   let doc = Pandoc mempty rest
   !summary <- first MarkdownFormatError $ parseAdvisorySummary doc
-  table <- firstPretty TomlError TOML.renderTOMLError $ TOML.decode frontMatter
+  table <- case Toml.parse (T.unpack frontMatter) of
```

Comment:
I'm also curious what is the performance impact of switching from a library that works with `Text` to a library that works on `String`.  Are there any benchmarks available?  Probably not a show-stopper, but it would be nice to know.

Reply:
I have no benchmarks for this. My expectation is that no one will be using a large enough TOML file to benefit from worrying about the performance difference here. I'll defer to your understanding of the hsec-tools use case on this, though.